plotter/barchart_test.go

```diff
@@ -0,0 +1,178 @@
+// Copyright ©2015 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package plotter
+
+import (
+	"image/color"
+	"testing"
+
+	"github.com/gonum/plot"
+	"github.com/gonum/plot/vg"
+)
+
+func ExampleBarChart() {
+	// Create the plot values and labels.
+	values := Values{0.5, 10, 20, 30}
+	verticalLabels := []string{"A", "B", "C", "D"}
+	horizontalLabels := []string{"Label A", "Label B", "Label C", "Label D"}
+
+	// Create a vertical BarChart
+	p1, err := plot.New()
+	handleEx(err)
```

Comment:
I'm really not a fan of using error handling functions for character count reduction at the best of times, but especially not in example code when the function is trivial and definition is not available to the reader.